projects/angular/src/emphasis/_variables.label.scss

```
@@ -6,57 +6,66 @@
-$clr-label-danger-font-color: variables.$clr-color-danger-900 !default;
-$clr-label-danger-border-color: variables.$clr-color-danger-900 !default;
+$clr-label-bg-hover-color: var(--clr-label-bg-hover-color) !default;
+
+$clr-label-gray-bg-color: var(--clr-label-gray-bg-color) !default;
+$clr-label-gray-border-color: var(--clr-label-gray-border-color) !default;
+$clr-label-gray-color: var(--clr-label-gray-color) !default;
+
+$clr-label-purple-bg-color: var(--clr-label-purple-bg-color) !default;
+$clr-label-purple-border-color: var(--clr-label-purple-border-color) !default;
+$clr-label-purple-color: var(--clr-label-purple-color) !default;
+
+$clr-label-blue-bg-color: var(--clr-label-blue-bg-color) !default;
+$clr-label-blue-border-color: var(--clr-label-blue-border-color) !default;
+$clr-label-blue-color: var(--clr-label-blue-color) !default;
+
+$clr-label-orange-bg-color: var(--clr-label-orange-bg-color) !default;
+$clr-label-orange-border-color: var(--clr-label-orange-border-color) !default;
+$clr-label-orange-color: var(--clr-label-orange-color) !default;
+
+$clr-label-light-blue-bg-color: var(--clr-label-light-blue-bg-color) !default;
+$clr-label-light-blue-border-color: var(--clr-label-light-blue-border-color) !default;
+$clr-label-light-blue-color: var(--clr-label-light-blue-color) !default;
+
+$clr-label-info-bg-color: var(--clr-label-info-bg-color) !default;
+$clr-label-info-font-color: var(--clr-label-info-font-color) !default;
+$clr-label-info-border-color: var(--clr-label-info-border-color) !default;
+
+$clr-label-success-bg-color: var(--clr-label-success-bg-color) !default;
+$clr-label-success-font-color: var(--clr-label-success-font-color) !default;
+$clr-label-success-border-color: var(--clr-label-success-border-color) !default;
+
+$clr-label-warning-bg-color: var(--clr-label-warning-bg-color) !default;
+$clr-label-warning-font-color: var(--clr-label-warning-font-color) !default;
+$clr-label-warning-border-color: var(--clr-label-warning-border-color) !default;
+
+$clr-label-danger-bg-color: var(--clr-label-danger-bg-color) !default;
+$clr-label-danger-font-color: var(--clr-label-danger-font-color) !default;
+$clr-label-danger-border-color: var(--clr-label-danger-border-color) !default;
+
+// New variables
+$clr-label-border-width: var(--clr-label-border-width) !default;
+$clr-label-height: var(--clr-label-height) !default;
+$clr-label-hover-color: var(--clr-label-hover-color) !default;
+$clr-label-active-color: var(--clr-label-active-color) !default;
+$clr-label-bg-active-color: var(--clr-label-bg-active-color) !default;
+$clr-label-box-shadow: var(--clr-label-box-shadow) !default;
+$clr-label-bg-disabled-color: var(--clr-label-bg-disabled-color) !default;
+$clr-label-disabled-color: var(--clr-label-disabled-color) !default;
+$clr-label-disabled-border-color: var(--clr-label-disabled-border-color) !default;
+$clr-label-disabled-color: var(--clr-label-disabled-color) !default;
```

What new variables mean. If we add more variables down the road will they became new new variables?
Order variables by in some meaningful way. Example:
 ```suggestion
// ACTION COLORS
$clr-label-hover-color: var(--clr-label-hover-color) !default;
$clr-label-active-color: var(--clr-label-active-color) !default;
$clr-label-bg-active-color: var(--clr-label-bg-active-color) !default;
$clr-label-box-shadow: var(--clr-label-box-shadow) !default;
$clr-label-bg-disabled-color: var(--clr-label-bg-disabled-color) !default;
$clr-label-disabled-color: var(--clr-label-disabled-color) !default;
$clr-label-disabled-border-color: var(--clr-label-disabled-border-color) !default;
$clr-label-disabled-color: var(--clr-label-disabled-color) !default;

// APPEARANCE
$clr-label-border-width: var(--clr-label-border-width) !default;
$clr-label-height: var(--clr-label-height) !default;
```